leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -465,7 +503,8 @@ var ElementInspector = Backbone.View.extend({
             }
         }
 
-        this.updateChart(null);
+        this.updateChart(null); 
```

Comment:
You never need to pass anything into updateChart, and in fact can remove the event parameter from that function as well because it is never used for anything